easy_my_coop_website/controllers/main.py

```diff
@@ -249,6 +252,8 @@ def validation(  # noqa: C901 (method too complex)
         sub_req_obj = request.env["subscription.request"]
 
         redirect = "easy_my_coop_website.becomecooperator"
+        # redirect url to fall back on become coopererator in template redirection
+        values["redirect_url"] = request.httprequest.host_url + "become_cooperator"
```

Comment:
I think it's safer to use `from urllib.parse import urljoin` to join url : it makes sure the `/` are present and not doubled among other things.

Reply:
Yes I agree, it's fixed :)